pkg/providers/amifamily/bootstrap/bottlerocket.go

```diff
@@ -64,6 +64,9 @@ func (b Bottlerocket) Script() (string, error) {
 		if b.KubeletConfig.EvictionHard != nil {
```

Comment:
Do you mind making an update to our functional and E2E testing to ensure that passing this value through works properly for BR?

Reply:
hello 👋🏽  I updated an existing test that merges the user-data content for BR family. Please let me know if this would suffice